src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -921,15 +927,13 @@ public void updateAllContentSelectors() {
         }
     }
 
-    /**
-     * Update the JTextArea when a field has changed.
-     *
-     * @see java.beans.VetoableChangeListener#vetoableChange(java.beans.PropertyChangeEvent)
-     */
-    @Override
-    public void vetoableChange(PropertyChangeEvent e) {
-        String newValue = e.getNewValue() == null ? "" : e.getNewValue().toString();
-        setField(e.getPropertyName(), newValue);
+    private class ChangeFieldListener {
```

Comment:
Why not keeping the old JavaDoc comment "Update the JTextArea when a field has changed."?